server/events/project_command_builder.go

```diff
@@ -223,9 +225,9 @@ func (p *DefaultProjectCommandBuilder) buildProjectPlanCommand(ctx *CommandConte
 	return p.buildProjectCommandCtx(ctx, models.PlanCommand, cmd.ProjectName, cmd.Flags, repoDir, repoRelDir, workspace, cmd.Verbose)
 }
 
-// buildApplyAllCommands builds apply contexts for every project that has
+// buildCommandsFromPlanFiles builds contexts for any command for every project that has
 // pending plans in this ctx.
-func (p *DefaultProjectCommandBuilder) buildApplyAllCommands(ctx *CommandContext, commentCmd *CommentCommand) ([]models.ProjectCommandContext, error) {
+func (p *DefaultProjectCommandBuilder) buildCommandsFromPlanFiles(ctx *CommandContext, cmdName models.CommandName, commentCmd *CommentCommand) ([]models.ProjectCommandContext, error) {
```

Comment:
Is it possible to modify. this to not be a receiver method?

If DefaultProjectCommandBuilder doesn't build policy check commands, then it seems like it shouldn't own this function that builds all commands.

Reply:
Also what's the motivation behind the naming of this function

`buildCommandsFromPlanFiles` isn't intuitive to me. Something like `buildProjectCommands` seems to make more intuitive sense.